SECURITY.md

```diff
@@ -8,9 +8,10 @@ Eclipse RDF4J supports security updates for the following releases:
 
```

Comment:
- ✅ the most recent stable release
- ✅ the latest minor release _before_ the current one 
- ✅ the latest major release _before_ the current
- ❌ anything older

(for example if the latest release is 4.1, we support security patches for 4.1.x and 4.0.x releases, as well as for 3.7.x, but not for 3.6.x or older)